### DISCUSSION TITLE
Menambahkan Data Baris pada dataframe TimeSeries

### DISCUSSION THREAD

**Discussion by tu849035** [teamalgoritma/community#337] [Q&A] [answered]
Salam.

Saya ingin bertanya bagaimana caranya menambahkan baris pada dataframe TimeSeries. Seperti pada tampilan di bawah,
![image](https://user-images.githubusercontent.com/107348211/191955334-752e1480-e344-4fe7-99ed-ee9e86c03d50.png)
Saya ingin menambahkan baris pada kolom "transaction_date" mulai "2017-12-01 10:00:00".
Karena ingin saya sesuaikan dengan pertanyaan bisnis untuk prediksi jumlah pengunjung pada toko yang mulai buka dari jam 10 pagi hingga jam 10 malam.
Saya sudah mencoba dengan fungsi 'pad()', tapi fungsi ini hanya menambahkan baris yg ada di antaranya saja, tidak menambahkan pada baris sebelumnya. Sedangkan saya  ingin menambahkan baris sebelum baris yg paling pertama agar bisa sesuai dengan pertanyaan bisnis.

Mohon bantuan saran & solusinya.

Terima Kasih.

**Comment by RanggaGemilang** [comment, marked as answer]
Izin bantu menjawab mas, mungkin bisa ditambahkan secara manual dengan cara seperti ini 
![datazet](https://user-images.githubusercontent.com/107049922/192183076-1bbbc9eb-7f73-46c8-bab5-1823506af74c.png)

datetime pada gambar diatas sama dengan kolom transaction_date yang mas punya,

**Reply by tu849035** [reply]
Terima Kasih Mas Rangga. Sangat membantu.
